Source/Diagnostics/ReducedDiags/BeamRelevant.cpp

```diff
@@ -47,7 +47,7 @@ BeamRelevant::BeamRelevant (std::string rd_name)
     //       11: rms gamma
     //    12,13: emittance x,z
     //       14: charge
-    m_data.resize(15,0.0);
+    m_data.resize(15, amrex::Real(0.0));
```

Comment:
You should be able to do this, if you want to be more compact
```suggestion
    m_data.resize(15, 0.0_rt);
```